src/app/shared/table/table.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+
+import { TalentService } from '@talent/talent.service';
+import { AbstractListComponent } from '@shared/component/abstract-list.component';
+
+import { Talent } from '@shared/model/talent.model';
+import { SortOptions } from '@shared/service/model/sort-options.model';
+
+@Component({
+	selector: 'table',
+	templateUrl: './table.component.html',
+	styleUrls: ['./table.component.scss']
+})
+export class TableComponent {
+    private _sortBy: string;
+	private _sortOrder: number;
```

Comment:
indentation, for the entire file

Reply:
also, avoid using _variable names. they are used in javascript, indeed, but that's an old school way of saying "don't use this variable as it is private". in typescript, you do have private and public, so you don't need this convention